tests/cupyx_tests/fallback_mode_tests/test_fallback.py

```diff
@@ -147,3 +147,114 @@ def test_func_not_in_numpy(self):
 
         with self.assertRaises(AttributeError):
             func = fallback_mode.numpy.dummy  # NOQA
+
+
+@testing.gpu
+class TestNotifications(unittest.TestCase):
+
+    def test_1_seterr_geterr(self):
+
+        default = fallback_mode.geterr()
+        assert default == 'warn'
+
+        old = fallback_mode.seterr('ignore')
+        current = fallback_mode.geterr()
+        assert old == 'warn'
+        assert current == 'ignore'
+
+    def test_notification_ignore(self):
+
+        import contextlib
+        from io import StringIO
+
+        saved_stdout = StringIO()
+        with contextlib.redirect_stdout(saved_stdout):
+            fallback_mode.seterr('ignore')
+            res = fallback_mode.numpy.nanargmin([1, 2, 3])  # NOQA
```

Comment:
local variable 'res' is assigned to but never used.
```suggestion
            fallback_mode.numpy.nanargmin([1, 2, 3])
```

Reply:
Fix also other NOQAs.

Reply:
I already did. Making other changes and then will push the code.